content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
+Operators can choose what is best for their users, there is no requirement to deploy every component.
+To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/kubeflow-overview/">Kubeflow Overview</a> page.
+
+There are two pathways to get up and running with Kubeflow, you may either:
+1. Use a [packaged distribution](#packaged-distributions)
+1. Use the [manifests](#manifests) (advanced)
+
+<a id="packaged-distributions"></a>
+## Packaged distributions
+
+{{% alert color="warning" %}}
+Packaged distributions are developed and supported by their respective maintainers, the Kubeflow community does not currently endorse or certify any distribution.
+{{% /alert %}}
+
+<b>See the table below for a list of options and links to documentation:</b>
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Name</th>
+        <th>Maintainer</th>
+        <th>Platform</th>
+        <th>Link - Docs</th>
+        <th>Link - External Website</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Kubeflow on AWS</td>
+        <td>AWS</td>
```

Comment:
Amazon Web Services (AWS)

Reply:
Best practice is to define acronyms, even those that are very well known.

Reply:
I am treating the first column, as the actual "name" of the distribution, which is precisely "Kubeflow on AWS".

Reply:
I now realise you were referring to the "maintainers" column, I have changed this to "Amazon Web Services".